build.js

```diff
@@ -64,11 +64,7 @@
             name: 'global-deps',
```

Comment:
should create be false as it no longer has any output? (I'm not super familiar with r.js build system so I'm not 100%)